packages/e2e-tests/src/specs/wordpress/webStoriesBlock.js

```diff
@@ -41,9 +41,14 @@ const EMBED_BLOCK_CONTENT = `
 describe('Web Stories Block', () => {
   let stopRequestInterception;
   let removeErrorMessage;
+  let removeError404Message;
 
   beforeAll(async () => {
+    // TODO, look into why this error is happening.
```

Comment:
Creating a ticket for this would be good

Reply:
Fixed.